S3TransferUtility-Sample/Swift/Podfile

```diff
@@ -4,5 +4,6 @@ platform :ios, '9.0'
 use_frameworks!
 
 target 'S3TransferUtilitySampleSwift' do
+    pod 'AWSMobileClient', '~> 2.8.0'
```

Comment:
Nit: We might consider declaring a variable for version 
$awsVersion = '~> 2.8.0'
pod 'AWSMobileClient', $awsVersion
pod 'AWSS3', $awsVersion